modules/local/build_intervals/main.nf

```diff
@@ -1,22 +1,30 @@
 process BUILD_INTERVALS {
-    tag "$fasta_fai"
+    tag "$meta.id"
 
-    conda (params.enable_conda ? "anaconda::gawk=5.1.0" : null)
+    conda (params.enable_conda ? "bioconda::gawk=4.1.3" : null)
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/gawk:5.1.0' :
-        'quay.io/biocontainers/gawk:5.1.0' }"
+        'https://depot.galaxyproject.org/singularity/gawk:4.1.3--0':
+        'quay.io/biocontainers/gawk:4.1.3--1' }"
 
     input:
-    path fasta_fai
+    tuple val(meta), path(fasta_fai)
 
     output:
-    path "*.bed", emit: bed
+    tuple val(meta), path("${fasta_fai.baseName}.bed") , emit: bed
+    path "versions.yml"                                , emit: versions
 
     when:
     task.ext.when == null || task.ext.when
 
     script:
+    def args = task.ext.args ?: ''
```

Comment:
Do we need to define `args`?

Reply:
probably not, I re-did the module with `nf-core modules create` at some point because i was running out of ideas why it wasn't working.